ci/prow-build-test-qemu.sh

```diff
@@ -4,6 +4,8 @@ set -xeuo pipefail
 # entrypoint for PRs to this repo, as well as for PRs on other repos,
 # mainly coreos-assembler.  It assumes that `cosa init` has been run.
 
+REDIRECTOR_URL="https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/"
```

Comment:
I would probably have done this by adding e.g. `cosa buildprep --distro=rhcos` or so in cosa, then use it here.  

That said, this actually leads into a whole other huge detail, which is around the split between bootimages and the oscontainer update that is prominent in RHCOS.

In some cases, if we're intending to do a bootimage update, starting CI from the previous bootimage would make sense.  And that's a separate URL.

There's also the long running huge problem that we can (and often have because brew) push an oscontainer without updating the redirector/S3, so the data there can be incorrect, and what we'd really want to do is fetch the previous oscontainer here.

Anyways, none of that should block this, which is a good improvement.  Thanks for doing this!

Reply:
> I would probably have done this by adding e.g. `cosa buildprep --distro=rhcos` or so in cosa, then use it here.

I'll pursue this as a follow-up.

> That said, this actually leads into a whole other huge detail, which is around the split between bootimages and the oscontainer update that is prominent in RHCOS.
> 
> In some cases, if we're intending to do a bootimage update, starting CI from the previous bootimage would make sense. And that's a separate URL.

I think in the case here, where we are just testing changes to this repo, starting from the last version found on the redirector is sane enough. We can improve the CI around bootimage bumps to `openshift/installer` by building out CI jobs that verify cluster installs (https://issues.redhat.com/browse/GRPA-4447).

Additionally (something that I think I'll pursue in the near future), we should expand the CI job here to build the metal + ISO artifacts so we can run the `testiso` suite.  Getting us closer to having more confidence of upstream changes and downstream becomes a rebuild.

> There's also the long running huge problem that we can (and often have because brew) push an oscontainer without updating the redirector/S3, so the data there can be incorrect, and what we'd really want to do is fetch the previous oscontainer here.
> 
> Anyways, none of that should block this, which is a good improvement. Thanks for doing this!

We should be in a better position to avoid that scenario of pushed oscontainer/no S3 update since we've removed the Brew stage from the downstream job. I'll investigate pulling the oscontainer as part of this CI job.